fbchat/_client.py

```diff
@@ -2727,6 +2727,17 @@ def getThreadIdAndThreadType(msg_metadata):
                 msg=delta,
             )
 
+        # New pending message
+        elif delta_class == "ThreadFolder" and delta.get("folder") == "FOLDER_PENDING":
+            threadId = delta["threadKey"]["otherUserFbId"]
```

Comment:
Use `getThreadIdAndThreadType`, since that handles group threads as well. But perhaps it's not important, I don't know if you can have pending group threads?

Reply:
Oh, and use snake case `thread_id` over camelCase `threadId`